pallets/bucket-nfts/Cargo.toml

```diff
@@ -16,6 +16,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { workspace = true }
+num-bigint = { workspace = true }
```

Comment:
If you only need this for the `mock.rs` file, please add them as `dev-dependencies`.